### PR TITLE
build(deps): update dependencies to match latest x-components version

### PR DESCRIPTION
**Pull request by tiborux**
EX-5883

### CODE REVIEW

**Review by LuisMartinez15 — DISMISSED** [review]

File: package.json
Comment (javieri-empathy) [inline package.json:21]: So for what I saw previously there were some files that were not lined right? Good catch

File: package.json
Comment (javieri-empathy) [inline package.json:20]:
```suggestion
```

This is no longer installed or used.

**Review by javieri-empathy — CHANGES_REQUESTED** [review]
The API does not work atm, but looks pretty good for me. No issues when compiling or running the dev server.

File: package.json
Comment (tiborux) [reply package.json:21]: yes, I copied the one from x-components because here we were not checking all of them.

File: package.json
Comment (LuisMartinez15) [inline package.json:31]: Can we update to the latest x-components version? Specially if we wait until [this pull request](https://github.com/empathyco/x/pull/454) gets merged, as it makes some stuff from the search `API` optional, fixing the current broken status due to the changes made in it.

**Review by LuisMartinez15 — CHANGES_REQUESTED** [review]
Can we update to the latest x-components version? Specially if we wait until this pull request gets merged, as it makes some stuff from the search API optional, fixing the current broken status due to the changes made in it.

**Review by LuisMartinez15 — APPROVED** [review]
GOOD JOB!!!!

**Pull request merged**
